src/AppBundle/Controller/TodoController.php

```diff
@@ -0,0 +1,242 @@
+    }
+
+    /**
+     * @Route("/todo", name="post_todo")
+     * @Method({"POST"})
+     *
+     * Creates a new to do, and responds with the ID value (string)
+     *
+     * @param Request $request
+     * @return JsonResponse
+     *
+     * request
+     *  body:
+     *      type: object
+     *      properties:
+     *          title:
+     *              type: string
+     *              required: true
+     *              description: The new to do item. Must not be empty.
+     * responses:
+     *  200:
+     *      OK
+     *  400:
+     *      Bad Request
+     *
+     *
+     */
+    public function postTodoAction(Request $request)
+    {
+        $body = null;
+        try {
+            $body = $this->bodyAsJSON($request);
+            //If title is unset or null, or isnt a string
+            if (!isset($body['title']) || !is_string($body['title']))
+                throw new JSONControllerException("Required parameter 'title' was unset or not a string value.");
+        } catch (JSONControllerException $e) {
+            return new JsonResponse('' . $e, 400);
+        }
+        //$body isnt null and title is set
+
+        $todo = new Todo();
+        $todo->setCompleted(false);
+        $todo->setTitle($body['title']);
+        $todo->setUserId($this->getUser()->getIdString());
+
+        $this->dm()->persist($todo);
+        $this->dm()->flush();
+
+        return new JsonResponse('' . $todo->getId());
+    }
+
+    /**
+     * @Route("/todo/{id}", name="get_todo_id")
+     * @Method({"GET"})
+     *
+     * Gets a specific item.
+     *
+     * @param string $id
+     * @return JsonResponse
+     */
+    public function getTodoIdAction($id)
+    {
+        $item = $this->get('doctrine_mongodb')
+            ->getRepository('AppBundle:Todo')
+            ->find($id);
+
+        if (!$item) {
+            return new JsonResponse('Not Found', 404);
+        }
+
+        //if we are forbidden (we are authorized)
+        if ($item->getUserId() != $this->getUser()->getIdString()) {
+            return new JsonResponse('Forbidden', 403);
+        }
+
+        return new JsonResponse($item->serialize());
+    }
+
+    /**
+     * @Route("/todo/{id}", name="delete_todo_id")
+     * @Method({"DELETE"})
+     *
+     * @param string $id
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function deleteTodoIdAction($id, Request $request)
+    {
+        $item = $this->get('doctrine_mongodb')
+            ->getRepository('AppBundle:Todo')
+            ->find($id);
+
+        if (!$item) {
+            return new JsonResponse('Not Found', 404);
+        }
+
+        //if we are forbidden (we are authorized)
+        if ($item->getUserId() != $this->getUser()->getIdString()) {
+            return new JsonResponse('Forbidden', 403);
+        }
+
+        $this->dm()->remove($item);
+        $this->dm()->flush();
+
+        return new JsonResponse('Successfully deleted ' . $id);
```

Comment:
It makes it easier to parse in the frontend if you send this as an array:
['message' => 'Sucessfully deleted', 'id' => $id]
Otherwise you'll have to do string manipulation to find the id that was deleted.